02_configure_host.sh

```diff
@@ -54,6 +54,16 @@ if [[ ! -z "${MIRROR_IMAGES}" || $(env | grep "_LOCAL_IMAGE=")  || ! -z "${ENABL
     setup_local_registry
 fi
 
+if [[ ! -z "${SQUID_PROXY}" ]]; then
+  generate_squid_conf > squid.conf
+
+  sudo podman run -d --rm \
+     --net host \
+     --volume $PWD/squid.conf:/etc/squid/squid.conf \
+     --name squid \
+     sameersbn/squid:latest
```

Comment:
```suggestion
     docker.io/sameersbn/squid:3.5.27-2
```
Podman takes less time when you specify the full path. Also, pointing to `latest` is a bit dangerous in case they push an update that needs newer configuration.

Also, I forget if we want to allow the host to be rebooted and keep the cluster working. In which case, this and other containers we run with podman, should be started with podman and then create a user systemd service with `podman generate`.